map/map_tests/CMakeLists.txt

```diff
@@ -19,10 +19,11 @@ set(SRC
   working_time_tests.cpp
 )
 
-omim_add_test_with_qt_event_loop(${PROJECT_NAME} ${SRC})
-
-target_link_libraries(${PROJECT_NAME}
-  search_tests_support
-  generator_tests_support
-  map
+omim_add_test(
+  NAME ${PROJECT_NAME}
+  SRC ${SRC}
+  REQUIRE_QT
+  REQUIRE_SERVER
 )
+
+target_link_libraries(${PROJECT_NAME} search_tests_support generator_tests_support map)
```

Comment:
Be consistent here. You call omim_add_test with _column_, but rewrite target_link_libraries with _line_.
Keep like it was before?